src/enclave/enclavetypes.h

```diff
@@ -34,6 +34,8 @@ namespace enclave
     };
     struct request req;
 
+    bool is_create_request = false;
```

Comment:
I think we could remove this altogether if (as well as moving the endpoint to `nodefrontend.h`), in `verify_client_signature()` (`frontend.h`), we return `true` if `!client_signatures`.

Reply:
discussed offline